### PR TITLE
CS-1127 Create Deployment Guide and CloudFormation Template

### PR DESCRIPTION
**Pull request by rogeryang-sugarcrm**

### CODE REVIEW

**Review by jimmychen — DISMISSED** [review]

File: assets/deliverables/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+This directory holds the plain CloudFormation template and instructions to package assets for customer installation without the need of AWS SAM.
```
Comment (sugarhqi) [inline assets/deliverables/README.md:1]: This is basically a build instruction. We probably want to create a build script to automate this process in the future. Should we include Deployment_Guide.pdf here instead?

File: assets/deliverables/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+These assets will be zipped into SugarLive_Features.zip with the following structure:
+
+```
+- SugarLive_Features.zip
+  - Deployment_Guide.pdf
+  - CloudFormation_Template.yml
+  - Lambda.zip
+  - Portal_Chat.zip
+    - Lex_Bot.zip
+    - SugarCRM_Portal_Chat_Contact_Flow-v1.0.json.json
```
Comment (sugarhqi) [inline assets/deliverables/README.md:25]: .json.json

Comment (rogeryang-sugarcrm) [reply assets/deliverables/README.md]: Fixed

File: assets/deliverables/README.md (hunk 1 above)
Comment (rogeryang-sugarcrm) [reply assets/deliverables/README.md:1]: Answered this on the [ticket](https://sugarcrm.atlassian.net/browse/CS-1127?focusedCommentId=806559)

**Review by sugarhqi — APPROVED** [review]

**Pull request merged**